pr.ts

```diff
@@ -26,16 +26,20 @@ export const changelog = wrap("Require changelog entries on PRs with code change
   const getContentParams = { path: "", owner: pr.head.user.login, repo: pr.head.repo.name }
   const rootContents: any = await danger.github.api.repos.getContent(getContentParams)
 
-  const hasChangelog = rootContents.data.find(file => changelogs.includes(file.name))
-  if (hasChangelog) {
+  const hasChangelog = rootContents.data.find((file: any) => changelogs.includes(file.name))
+  const markedTrivial = pr.title.match("#trivial")
+  if (hasChangelog && !markedTrivial) {
     const files = [...danger.git.modified_files, ...danger.git.created_files]
 
     // Look for Swift files that aren't in a unit test directory.
-    const hasCodeChanges = files.find(file => file.match(/.*\.swift/) && !file.match(/(test|spec)/i))
+    const hasCodeChanges = files.find((file: any) => file.match(/.*\.swift/) && !file.match(/(test|spec)/i))
     const hasChangelogChanges = files.find(file => changelogs.includes(file))
 
     if (hasCodeChanges && !hasChangelogChanges) {
-      warn("It looks like code was changed without adding anything to the Changelog.")
+      warn(
+        "It looks like code was changed without adding anything to the Changelog." +
+          "If this is a trivial PR that doesn't need a changelog, add #trivial to the PR title."
```

Comment:
Just wondering, would a title change be enough for Peril to run again? or would you need CI to run again? 

Or actually, I guess Peril doesn't even rely on a CI, it's its own step here, right?

Reply:
Also, mega-super-uber-nit, there is an extra space on the second appended line (unless this is some TS convention I don't know 😛)

Reply:
That's right – Peril should get webhook notifications for changed PR titles, but I'm not sure if it re-runs the danger file (I expect it does but couldn't confirm looking through Peril's code). @orta can you confirm?

Reply:
It's fine if it doesn't re-run the Danger file, was just asking if Peril execute again when the title changes, or only when the PR is initially created. Seems like it's the former, which is great 👍

Reply:
Any update to the PR, so a new body, a new commit, opening/closing, adding new assignees
 etc - see http://artsy.github.io/blog/2017/09/04/Introducing-Peril/#Events

https://github.com/RxSwiftCommunity/peril/blob/d4dee0132368def87b2ecb6cf45c54e2df1e0304/settings-peril.json#L6